assemble/bin/accumulo-cluster

```diff
@@ -28,18 +28,137 @@ Options:
 Commands:
   create-config              Creates cluster config
   restart                    Restarts the Accumulo cluster
```

Comment:
`restart` could support the same options as start, stop, and kill. This PR just retains the existing behavior of restarting everything, but that could be easily changed.

Reply:
Implemented this in 8ee5214